databuilder/__main__.py

```diff
@@ -61,8 +60,6 @@ def main(args=None):
             backend=options.backend,
             url=options.url,
         )
-    elif options.which == "generate_docs":
-        generate_docs(location=options.location)
```

Comment:
No idea, sorry -- @ghickman?

Reply:
We'll need to change the justfile for [the tooling there](https://github.com/opensafely/documentation/blob/main/databuilder/justfile#L132-L133).  So I think this is fine because we can mirror what's been put into the justfile for these changes, we'll just need to make sure we point the output at the right location, but it should be a case of `python -m databuilder.docs > ../public_docs.json` 🤞

Reply:
Thanks!